src/System.Reflection/tests/FieldInfoTests.cs

```diff
@@ -540,12 +545,14 @@ struct FieldData
         [InlineData(null)]
         public static void SetValueDirect_GetValueDirectRoundDataTest(object value)
         {
-            FieldData testField = new FieldData { field = -1 };
+            FieldData testField = new FieldData { inner = new Inner() { field = -1 } };
+            FieldInfo innerFieldIinfo = typeof(FieldData).GetField(nameof(FieldData.inner));
```

Comment:
Nit: I assume this is a typo and it should be innerFieldInfo rather than innerFieldIinfo?